loops_lab.js

```diff
@@ -0,0 +1,109 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Good demonstration of for and while loops overall. Other than minor feedback mentioned below, no major issues. Good job and keep up the good work!